examples/vuepress/.gitignore

```diff
@@ -0,0 +1,12 @@
+pids
```

Comment:
You might want to combine this with the other `.gitignore` files in this repo, which add some Vercel specific stuff 👍